### PR TITLE
feat: implemented hooks for interest rate

### PR DESCRIPTION
**Pull request by k3nt0w**
## Proposed Changes

- As with the title

## Implementation

- hooks wrapper for calculating interest rate

## Remaining Task

- `aggregate` doesn't work

### CODE REVIEW

**Comment by k3nt0w** [comment]
@aggre 
The keyword `_aggregate` doesn't seem to work in the console of `hasura`. I need an environment that can run GraphQL with `_aggregate` keyword, could you check it?

**Comment by aggre** [comment]
Try deleting the `x-hasura-role`. @k3nt0w

**Comment by k3nt0w** [comment]
@aggre 
I'm sorry, it's in the documentation. Thank you very much.

File: packages/web/src/fixtures/utility/gql-hooks-wrapper.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { useGetRewardCalculationResultAggregateQuery } from '@dev/graphql'
+
+export const useAverageInterestRate = (metrics: string) => {
+  const { data: rewardCalculationResultData } = useGetRewardCalculationResultAggregateQuery({
+    variables: { metricsList: [metrics] }
+  })
+  const lockup = rewardCalculationResultData?.reward_calculation_result_aggregate.aggregate?.sum?.lockup
+  const stakingReward = rewardCalculationResultData?.reward_calculation_result_aggregate.aggregate?.sum?.staking_reward
+  return lockup && stakingReward ? lockup / stakingReward : 0
```
Comment (aggre) [inline packages/web/src/fixtures/utility/gql-hooks-wrapper.ts:9]: Sorry! I mistook the formula. The correct formula is `stakingReward / lockup`.

**Pull request merged**